aws-alb-ingress-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,130 @@
+{{ $root := . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "aws-alb-ingress-controller.fullname" $root }}
+  labels: {{ include "aws-alb-ingress-controller.labels" $root | nindent 4 }}
+    {{- range $key, $value :=.Values.deployment.labels }}
+    {{ $key }}: {{ tpl $value $root | quote }}
+    {{- end }}
+  {{- range $key, $value := .Values.deployment.annotations }}
+  annotations:
+    {{ $key }}: {{ tpl $value $root | quote }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "aws-alb-ingress-controller.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels: {{ include "aws-alb-ingress-controller.instance-labels" $root | nindent 8 }}
+        {{- range $key, $value := .Values.podLabels }}
+        {{ $key }}: {{ tpl $value $root | quote }}
+        {{- end }}
+        {{- range $key, $value := .Values.podAnnotations }}
+      annotations:
+        {{ $key }}: {{ tpl $value $root | quote }}
```

Comment:
If `$value` is a multi-line string, this will not work.
Since `alb.ingress.kubernetes.io/listen-ports` may contain multi-line string, please change it as follows.

```
{{ $key }}: {{ tpl $value $root | toYaml | indent 8 }}
```

Reply:
I fixed it because one line had indents.

```
        annotations:
-         cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+         cluster-autoscaler.kubernetes.io/safe-to-evict:         "true"
```